SU-SOE/soe_basic
# D8CORE-2338: typography tweaks for soe

# READY FOR REVIEW

# Summary
- Updates to the Cta list paragraph

# Needed By (Date)
- Tuesday, 7th

# Urgency
- Med

# Steps to Test

1. Pull in this branch
2. Verify all the headings in a WYSIWYG text area are font-weight 900. The splash is also 900.

# Affected Projects or Products
- SOE subtheming

# Associated Issues and/or People
-  D8CORE-2338

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
